common/src/main/java/org/broadleafcommerce/common/email/service/message/VelocityMessageCreator.java

```diff
@@ -49,7 +49,7 @@ public String buildMessageBody(EmailInfo info, Map<String,Object> props) {
             if (additionalConfigItems != null) {
                 propsCopy.putAll(additionalConfigItems);
             }
-            return VelocityEngineUtils.mergeTemplateIntoString(velocityEngine, info.getEmailTemplate(), info.getEncoding(), propsCopy);
+            //return VelocityEngineUtils.mergeTemplateIntoString(velocityEngine, info.getEmailTemplate(), info.getEncoding(), propsCopy);
```

Comment:
think we need to do something else here ? return formatted string ?
check https://stackoverflow.com/questions/14314143/velocityengineutils-has-been-removed-in-spring-3-2-so-what-else-to-use